swig/java/CMakeLists.txt

```diff
@@ -11,79 +11,97 @@ include(UseJava)
 include_directories(${JAVA_INCLUDE_PATH})
 include_directories(${JNI_INCLUDE_DIRS})
 
-if (DISABLE_SWIG OR NOT SWIG_EXECUTABLE)
-	file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/interface/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-	if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
-		# Update if necessary
-		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-long-long")
-	endif()
-	add_library(JNIhelics SHARED interface/helicsJavaJAVA_wrap.c)
-	target_link_libraries(JNIhelics helicsSharedLib ${JAVA_LIBRARIES})
-	set_target_properties (JNIhelics PROPERTIES FOLDER interfaces)
+if(DISABLE_SWIG OR NOT SWIG_EXECUTABLE)
+  file(COPY
+       ${CMAKE_CURRENT_SOURCE_DIR}/interface/
+       DESTINATION
+       ${CMAKE_CURRENT_BINARY_DIR})
+  if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+    # Update if necessary
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-long-long")
+  endif()
+  add_library(JNIhelics SHARED interface/helicsJavaJAVA_wrap.c)
+  target_link_libraries(JNIhelics helicsSharedLib ${JAVA_LIBRARIES})
+  set_target_properties(JNIhelics PROPERTIES FOLDER interfaces)
 else(DISABLE_SWIG)
-	#Enable generation using swig
+  # Enable generation using swig
 
-	set_property(SOURCE helicsJava.i PROPERTY SWIG_MODULE_NAME JNIhelics)
+  set_property(SOURCE helicsJava.i PROPERTY SWIG_MODULE_NAME JNIhelics)
 
-	set(CMAKE_SWIG_FLAGS "-package;com.java.helics")
-	if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER 3.7)
-		SWIG_ADD_LIBRARY(JNIhelics TYPE MODULE LANGUAGE java SOURCES helicsJava.i)
-	else ()
-		SWIG_ADD_MODULE(JNIhelics java helicsJava.i)
-	endif ()
+  set(CMAKE_SWIG_FLAGS "-package;com.java.helics")
+  if(${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER 3.7)
+    swig_add_library(JNIhelics TYPE MODULE LANGUAGE java SOURCES helicsJava.i)
+  else()
+    swig_add_module(JNIhelics java helicsJava.i)
+  endif()
 
-	SWIG_LINK_LIBRARIES(JNIhelics helicsSharedLib)
-	SWIG_LINK_LIBRARIES(JNIhelics ${JAVA_LIBRARIES})
+  swig_link_libraries(JNIhelics helicsSharedLib)
+  swig_link_libraries(JNIhelics ${JAVA_LIBRARIES})
 
```

Comment:
Does capitalization matter here for these functions

Reply:
I think in CMake `commands` are case insensitive but arguments are case sensitive.

Reply:
I don't think I realized that was true for user defined functions as well as the built in ones.